lightning/src/ln/chan_utils.rs

```diff
@@ -255,6 +255,21 @@ pub fn get_htlc_redeemscript(htlc: &HTLCOutputInCommitment, keys: &TxCreationKey
 	get_htlc_redeemscript_with_explicit_keys(htlc, &keys.a_htlc_key, &keys.b_htlc_key, &keys.revocation_key)
 }
 
+/// redeem script for funding transaction output
```

Comment:
Docs should be clearer on something exposed publicly. Maybe "Gets the redeemscript for a funding output from the two funding public keys. Note that the order of funding public keys does not matter." (also maybe change to a and b given the order is irrelevant).

Reply:
used your text, thank you